peter_rockwood_HW5/cards4-template.js

```diff
@@ -0,0 +1,133 @@
+var makeCard = // receive factory with external name `makeCard`
```

Comment:
Looks good throughout! I saw some console logs indicating some problems that you seem to have fixed. Let me know if I missed anything that's still a problem, and we can take a look at it.